openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/TagSuggestion.tsx

```diff
@@ -82,6 +85,7 @@ const TagSuggestion: React.FC<TagSuggestionProps> = ({
     <AsyncSelectList
       defaultValue={value?.map((item) => item.tagFQN) || []}
       fetchOptions={isGlossaryType ? fetchGlossaryList : fetchTagsElasticSearch}
+      initialData={initialData}
```

Comment:
```suggestion
      initialOptions={initialOptions}
```

Reply:
@chirag-madlani , cool, that would be more helpful, thanks!